changelogutils/reader_test.go

```diff
@@ -51,7 +64,7 @@ var _ = Describe("ReaderTest", func() {
 		})
 
 		It("can read changelog file", func() {
-			changelogFile, err := reader.ReadChangelogFile(ctx, "changelog/v0.1.1/1.yaml")
+			changelogFile, err := reader.ReadChangelogFile(ctx, fmt.Sprintf("changelog/v0.1.1/%s", log))
```

Comment:
Since I found a similar v0.1.1, I'm using Sprintf to select the filename -- I could have left the entire string in the variables above, but this seemed slightly easier to understand. (Apparently `changelog/{rev}/` is a thing.)